src/main/java/io/openliberty/tools/common/plugins/util/PrepareFeatureUtil.java

```diff
@@ -114,11 +116,12 @@ private void prepareFeature(String groupId, String artifactId, String version) t
 				Map<File, String> esaFiles = downloadArtifactsFromBOM(additionalBOM);
 				File generatedJson = generateJson(targetJsonFile, esaFiles);
 				if (generatedJson.exists()) {
-					info("Json has been generated at the following location: " + generatedJson);
+					jsonFile = generatedJson;
+					info("Features.json has been generated at the following location: " + generatedJson);
 				}
 			} catch (PluginExecutionException e) {
-				warn("unable to find BOM file for given groupId " + groupId);
-				error("expecting BOM file artifact to be in repository: " + e.getMessage());
+				error(e.getMessage());
+				warn("Provide feautres-bom file at the given groupId " + groupId);
```

Comment:
Is this instructing the user to do something? Perhaps "A features-bom file must be provided at the given groupId " + groupdId "."?